cli/pyconv.ts

```diff
@@ -1739,15 +1739,19 @@ export function convertAsync(fns: string[]) {
         }
     }
 
+    const pkgFilesKeys = Object.keys(pkgFiles);
+    pxt.log(`files (${pkgFilesKeys.length}):\n   ${pkgFilesKeys.join('    \n')}`);
+
     return nodeutil.spawnWithPipeAsync({
-        cmd: "python3",
+        cmd: /^win/i.test(process.platform) ? "py" : "python3",
```

Comment:
If you have Python2 installed as well `py` will most likely resolve to the python2 installation. So if you definitely need to run a 3.x version perhaps `py -3` would be better. I would assume the `-3` flag will work even if you only have a single version of python 3.x installed, but perhaps it would be better to confirm that.